Sources/Sails/Application.swift

```diff
@@ -63,8 +65,10 @@ public class Application {
 }
 
 extension Application: Verifier {
-    public func verify(_ method: Method, _ uri: String) -> Int {
-        requestsMap[RequestMade(method: method, uri: uri), default: 0]
+    public func verify(_ method: Method, _ uri: String, times: UInt = 1, file: StaticString = #filePath, line: UInt = #line) {
+        if requestsMap[RequestMade(method: method, uri: uri), default: 0] < times {
+            XCTFail("Unable to verify request \(method) \(uri)\t", file: file, line: line)
+        }
```

Comment:
Looks good but, I was actually thinking that if we pass `times` then it would fail if the the request hasn't been sent that exact number of times. 

```swift
// This should fail if the request has been sent 1 or 3 times
server.verify(.get, "http://example.com", times: 2) 

// If we don't pass a value for `times` then it should pass as long as the request has been sent at least once
server.verify(.get, "http://example.com")
```

So I was thinking of an implementation like this:
```swift
    public func verify(_ method: Method, _ uri: String, times: UInt? = nil, file: StaticString = #filePath, line: UInt = #line) {
    let callCount = requestsMap[RequestMade(method: method, uri: uri), default: 0]
    if let times = times, times != callCount {
             XCTFail("Expected to receive request \(times) times but it was sent \(callCount) times")
             return
    }
    
    if callCount < 1 {
             XCTFail("Unable to verify request \(method) \(uri)\t", file: file, line: line)
    }
}
```

I'm writing this while in a zoom meeting so the above code examples are probably not perfect